File: fixtures/scale/csharp/medium/Eval/EvalError.cs
using System;

namespace Calc.Eval
{
    public class EvalError : Exception
    {
        public string Subject { get; }

        public EvalError(string message, string subject) : base(message)
        {
            Subject = subject;
        }

        public string Pretty()
        {
            return Message + ": " + Subject;
        }
    }
}

File: fixtures/scale/php/medium/src/Eval/EvalError.php
<?php

namespace Calc\Eval;

class EvalError extends \Exception
{
    public function __construct(string $message, public string $subject)
    {
        parent::__construct($message);
    }

    public function pretty(): string
    {
        return $this->getMessage() . ': ' . $this->subject;
    }
}
